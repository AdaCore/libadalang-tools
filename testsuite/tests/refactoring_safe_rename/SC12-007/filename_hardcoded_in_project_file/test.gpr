project Test is
   for Main use ("Test.2.ada");
   package Naming is
      for Casing               use "mixedcase";
      for Dot_Replacement      use "--";
      for Spec_Suffix ("Ada")  use ".1.ada";
      for Body_Suffix ("Ada")  use ".2.ada";
      for Spec ("Qux_1") use "qux.1.ada" at 1;
      for Spec ("Qux_2") use "qux.1.ada" at 2;
      for Spec ("Corge") use "corge.ads";
   end Naming;
end Test;
