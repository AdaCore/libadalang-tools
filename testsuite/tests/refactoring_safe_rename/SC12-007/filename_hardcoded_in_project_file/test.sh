#!/bin/sh
safe_rename  -P test.gpr -S corge.ads -L 1 -R 10 -N Fox --algorithm analyse_ast
