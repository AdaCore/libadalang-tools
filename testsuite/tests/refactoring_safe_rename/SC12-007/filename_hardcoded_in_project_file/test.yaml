description: |
    This test checks if the safe rename tool does not rename
    a hardcoded source filename in the the Naming package from
    project file.

driver: shell_script
