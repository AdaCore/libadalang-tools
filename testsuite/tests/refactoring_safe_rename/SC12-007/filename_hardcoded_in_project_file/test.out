# Renaming node corge.ads:1:9: 
References:
Unit:Test.2.ada
Line: 6
Source Location Range:
Start_Line: 6
Start_Column: 6
End_Line: 6
End_Column: 11
Unit:corge.ads
Line: 1
Source Location Range:
Start_Line: 1
Start_Column: 9
End_Line: 1
End_Column: 14
Line: 3
Source Location Range:
Start_Line: 3
Start_Column: 5
End_Line: 3
End_Column: 10
File renames:
corge.ads -> Fox.1.ada

